.github/dependabot.yml

```diff
@@ -2,8 +2,7 @@
 version: 2
 updates:
   # Maven
-  -
-    package-ecosystem: "maven"
+  - package-ecosystem: "maven"
```

Comment:
This needs to be replaced with gradle, right? Do you know if we have an issue for that already?

Reply:
> just a few comments already - hard to review if you are still pushing things 😅 Will try to review again at the weekend.

yeah, i'm sorry, just went through the code, and noticed a couple of things, which i pushed right away ... :/

Reply:
i don't think we haven issue, will create one

Reply:
[edit] #216